### PR TITLE
Updated AutoAdmittedUsers description in New/Set-CsTeamsMeetingPolicy.md

### PR DESCRIPTION
**Pull request by dariomws**
Related to https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7665.

### CODE REVIEW

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -322,7 +322,16 @@ Accept wildcard characters: False
 ```
 
 ### -AutoAdmittedUsers
-Determines what types of participants will automatically be added to meetings organized by this user. Set this to EveryoneInCompany if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately. Set this to Everyone if you'd like to admit anonymous users by default. Set this to EveryoneInSameAndFederatedCompany if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby.
+Determines what types of participants will automatically be added to meetings organized by this user.
+Possible values are:
+- EveryoneInCompany, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:327]:
```suggestion
- **EveryoneInCompany**, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately.
```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -322,7 +322,16 @@ Accept wildcard characters: False
...
+- EveryoneInSameAndFederatedCompany, if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:328]:
```suggestion
- **EveryoneInSameAndFederatedCompany**, if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby. 
```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 3]
```diff
@@ -322,7 +322,16 @@ Accept wildcard characters: False
 ```
 
 ### -AutoAdmittedUsers
-Determines what types of participants will automatically be added to meetings organized by this user. Set this to EveryoneInCompany if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately. Set this to Everyone if you'd like to admit anonymous users by default. Set this to EveryoneInSameAndFederatedCompany if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby.
+Determines what types of participants will automatically be added to meetings organized by this user.
+Possible values are:
+- EveryoneInCompany, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately.
+- EveryoneInSameAndFederatedCompany, if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby. 
+- Everyone, if you'd like to admit anonymous users by default.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:329]:
```suggestion
- **Everyone**, if you'd like to admit anonymous users by default. 
```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 4]
```diff
@@ -322,7 +322,16 @@ Accept wildcard characters: False
 ```
 
 ### -AutoAdmittedUsers
-Determines what types of participants will automatically be added to meetings organized by this user. Set this to EveryoneInCompany if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately. Set this to Everyone if you'd like to admit anonymous users by default. Set this to EveryoneInSameAndFederatedCompany if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby.
+Determines what types of participants will automatically be added to meetings organized by this user.
+Possible values are:
+- EveryoneInCompany, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately.
+- EveryoneInSameAndFederatedCompany, if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby. 
+- Everyone, if you'd like to admit anonymous users by default. 
+- OrganizerOnly, if you would like that only meeting organizers can bypass the lobby.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:330]:
```suggestion
- **OrganizerOnly**, if you would like that only meeting organizers can bypass the lobby.
```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 5]
```diff
@@ -322,7 +322,16 @@ Accept wildcard characters: False
...
+- EveryoneInCompanyExcludingGuests, if you would like meetings to place every external and guest users in the lobby but allow all other users in the company to join the meeting immediately.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:331]:
```suggestion
- **EveryoneInCompanyExcludingGuests**, if you would like meetings to place every external and guest users in the lobby but allow all other users in the company to join the meeting immediately.
```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 6]
```diff
@@ -322,7 +322,16 @@ Accept wildcard characters: False
...
+- InvitedUsers, if you would like that only meeting organizers and invited users can bypass the lobby.
```
Comment (tiburd) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:332]:
```suggestion
- **InvitedUsers**, if you would like that only meeting organizers and invited users can bypass the lobby.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 7]
```diff
@@ -337,10 +337,12 @@ Accept wildcard characters: False
 ### -AutoAdmittedUsers
 Determines what types of participants will automatically be added to meetings organized by this user.
 Possible values are:
-- EveryoneInCompany, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately. 
-- Everyone, if you'd like to admit anonymous users by default. 
+- EveryoneInCompany, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately.
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:340]:
```suggestion
- **EveryoneInCompany**, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 8]
```diff
@@ -337,10 +337,12 @@ Accept wildcard characters: False
...
 - EveryoneInSameAndFederatedCompany, if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby.
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:341]:
```suggestion
- **EveryoneInSameAndFederatedCompany**, if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby. 
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 9]
```diff
@@ -337,10 +337,12 @@ Accept wildcard characters: False
 ### -AutoAdmittedUsers
 Determines what types of participants will automatically be added to meetings organized by this user.
 Possible values are:
-- EveryoneInCompany, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately. 
-- Everyone, if you'd like to admit anonymous users by default. 
+- EveryoneInCompany, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately.
 - EveryoneInSameAndFederatedCompany, if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby. 
+- Everyone, if you'd like to admit anonymous users by default.
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:342]:
```suggestion
- **Everyone**, if you'd like to admit anonymous users by default. 
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 10]
```diff
@@ -337,10 +337,12 @@ Accept wildcard characters: False
 ### -AutoAdmittedUsers
 Determines what types of participants will automatically be added to meetings organized by this user.
 Possible values are:
-- EveryoneInCompany, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately. 
-- Everyone, if you'd like to admit anonymous users by default. 
+- EveryoneInCompany, if you would like meetings to place every external user in the lobby but allow all users in the company to join the meeting immediately.
 - EveryoneInSameAndFederatedCompany, if you would like meetings to allow federated users to join like your company's users, but place all other external users in a lobby. 
+- Everyone, if you'd like to admit anonymous users by default. 
 - OrganizerOnly, if you would like that only meeting organizers can bypass the lobby.
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:343]:
```suggestion
- **OrganizerOnly**, if you would like that only meeting organizers can bypass the lobby.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 11]
```diff
@@ -337,10 +337,12 @@ Accept wildcard characters: False
...
+- EveryoneInCompanyExcludingGuests, if you would like meetings to place every external and guest users in the lobby but allow all other users in the company to join the meeting immediately.
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:344]:
```suggestion
- **EveryoneInCompanyExcludingGuests**, if you would like meetings to place every external and guest users in the lobby but allow all other users in the company to join the meeting immediately.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 12]
```diff
@@ -337,10 +337,12 @@ Accept wildcard characters: False
...
+- InvitedUsers, if you would like that only meeting organizers and invited users can bypass the lobby.
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:345]:
```suggestion
- **InvitedUsers**, if you would like that only meeting organizers and invited users can bypass the lobby.
```

**Pull request merged**